formats.go

```diff
@@ -67,7 +67,35 @@ func writeEnvFile(filename string, data map[string]interface{}, mode os.FileMode
 	return writeFile(filename, buf.Bytes(), mode)
 }
 
+func writeCAChain(filename string, data map[string]interface{}, mode os.FileMode) error {
+	const element = "ca_chain"
+	const suffix = "ca"
+
+	// assume this is always a slice so cast the entry to []interface
+	chain, found := data[element].([]interface{})
+	if !found {
+		glog.Errorf("didn't find the certification option: %s", element)
+	}
```

Comment:
Yeah I think, `ok` is a better name. 
I believe that if the element doesn't exist it would return `nil` in which case the type assert would fail and `ok` would become false.